src/bilder/images/tika/deploy.py

```diff
@@ -145,6 +149,24 @@
 for product in hashicorp_products:
     configure_hashicorp_product(product)
 
+# Place the tags document
+tags_json = json.dumps(
+    build_tags_document(
+        source_tags={
+            "consul_version": VERSIONS["consul"],
+            "vault_version": VERSIONS["vault"],
+            "traefik_version": VERSIONS["traefik"],
```

Comment:
Should this also include the Tika version?

Reply:
So, that is in an interesting question/point and comes up in an issue I opened yesterday as I was working on this and doing my experimentation with Tika. https://github.com/mitodl/ol-infrastructure/issues/2033 The tika version we are using is actually hard coded at the moment, so while I could certainly include that hard coded value here I figured it made more sense to address it outright in its own issue and fix the greater problem.

Reply:
Makes sense, I'd rather get this merged and address in future work.